nix/package.nix

```diff
@@ -27,6 +27,10 @@
   revision ? "dirty",
   optimize ? "Debug",
   x11 ? true,
+  enableWayland ? true,
```

Comment:
I think we should ensure `x11` and `enableWayland` are using the same style. So we should probably change `x11` to `enableX11`. I'm not sure what is idiomatic for Nix here, but whatever is we should do.

Reply:
The nixpkgs package has `x11Support` so probably `waylandSupport`?

Reply:
It's usually withX11 or enableX11 in my experience

Reply:
I think pluie is a Nixpkgs maintainer, no? In that case, I trust your judgement more on this one. I don't care what the wording is as long as its idiomatic and consistent.

Reply:
Can confirm that camel casing and enable prefixes are commonly used to differentiate boolean options from packages.